src/shell/shell_cmds.cpp

```diff
@@ -252,21 +257,41 @@ void DOS_Shell::CMD_DELETE(char * args) {
 	dos.dta(save_dta);
 }
 
-void DOS_Shell::PrintHelpForCommands(const HELP_LIST requested_list)
+#define PRINT_HELP_CHECK_ROWS if (++rows_printed == nrows) { \
```

Comment:
This is a bit... ugh. Not sure of a nicer way to deal with it though.

Reply:
Why convert it to a lambda within the function?